<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the private 'console.command.messenger_failed_messages_remove' shared service.

include_once $this->targetDirs[3].'/vendor/symfony/console/Command/Command.php';
include_once $this->targetDirs[3].'/vendor/symfony/messenger/Command/AbstractFailedMessagesCommand.php';
include_once $this->targetDirs[3].'/vendor/symfony/messenger/Command/FailedMessagesRemoveCommand.php';

$this->privates['console.command.messenger_failed_messages_remove'] = $instance = new \Symfony\Component\Messenger\Command\FailedMessagesRemoveCommand('failed', ($this->privates['messenger.transport.failed'] ?? $this->load('getMessenger_Transport_FailedService.php')));

$instance->setName('messenger:failed:remove');

return $instance;
